### DISCUSSION TITLE
Hide modal full screen button

### DISCUSSION THREAD

**Discussion by benaunt** [Budibase/budibase#11580] [Help] [answered]
Hi,

Does anyone know of a way to hide the full screen option when using a modal?

We are using this to display some text a user needs to read through and need to restrict the user going to the underlying page the modal is from.

![image](https://github.com/Budibase/budibase/assets/83434591/daa8e289-6b70-4bcb-ba62-ba4eb1d9f68f)

Thanks

**Comment by ConorWebb96** [comment, marked as answer]
Hey @benaunt,

You can use the embed component along with a CSS hack to remove the full screen button. A warning though, this won't be limited to just this modal on the screen but all others you may have within this page.

```
<style>
  .modal-inner-wrapper .actions .btn-wrap:first-of-type {
    display: none;
  }
</style>
```
<img width="814" alt="Screenshot 2023-08-23 at 08 14 54" src="https://github.com/Budibase/budibase/assets/126772285/2f5b29f2-7dc3-4ba5-b000-15d17b5631fd">

<img width="652" alt="Screenshot 2023-08-23 at 08 16 56" src="https://github.com/Budibase/budibase/assets/126772285/74362cee-582c-4fd8-996d-d3ef10f33f67">

I hope this helps you.
